extra/mtls/docker-compose.mtls-test.yml

```diff
@@ -0,0 +1,23 @@
+version: '2.3'
+services:
+  mtls-gateway:
```

Comment:
see comment below, in my PR I was doing everything I can to not touch the test -- this is the way you prove that all works. you change the name to `mtls-gateway`, but not `mender-gateway`, easier to leave the original then?

Reply:
> see comment below, in my PR I was doing everything I can to not touch the test -- this is the way you prove that all works.

See reply above  :smiley:

> you change the name to `mtls-gateway`, but not `mender-gateway`, easier to leave the original then?

See reply above :smiley: